### DISCUSSION TITLE
Offensive comments received by mail, not visible in history ?

### DISCUSSION THREAD

**Discussion by lelegard** [community/community#61824] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hi all,

I own a few projects with many users and active discussions in issues. I receive an email each time something happens, specifically new comments in issues. 

Recently, two users apparently went in a fight, exchanging insults from one when the other opens an issue, and vice versa.  

I can see these messages in the notification emails. However, these offensive messages are not seen in the corresponding issue, either using a browser or the GitHub iOS app.

The offensive comments do not even appear as deleted in the issue history. It seems that these comments never existed. Except that I received them through emails. Maybe other users who subscribed to the project also received them. 

How is this possible ? How does this work ?

Clearly, it is a good thing that offensive comments are deleted but I want to understand what happens in my project underneath.

**Comment by moonfury-ops** [comment]
Thank you
